chain/chain.go

```diff
@@ -198,7 +198,7 @@ func (e *EthChainContext) Start() {
 
 	if len(e.Keys.Accounts()) < 1 {
 		if err := e.createAccount(); err != nil {
-			log.Error(err)
+			log.Errorf("cannot create ethereum account: %s", err)
```

Comment:
this looks like a Fatal

Reply:
Done, also capture the error on the account unlock. Fatal if cannot unlock account